x-pack/legacy/plugins/monitoring/public/components/elasticsearch/shard_allocation/_shard_allocation.scss

```diff
@@ -14,52 +14,57 @@ monitoring-shard-allocation {
   cluster-view {
     display: block;
   }
-  .parent {
-    padding-top: 14px;
-    border-left: 3px solid $euiColorSuccess !important;
-    &.red {
-      border-left: 3px solid $euiColorDanger !important;
-    }
-    &.yellow {
-      border-left: 3px solid $euiColorWarning !important;
-    }
-  }
-  td.unassigned {
+
+  .monUnassigned {
     vertical-align: middle;
     width: 150px;
   }
 
-  .child {
+  .monChildren {
+    padding-top: $euiSizeL;
+  }
+
+  .monChild {
     float: left;
     align-self: center;
+    background-color: $euiColorLightestShade;
+    margin: $euiSizeS;
+    border: 1px solid $euiColorMediumShade;
+    border-radius: $euiSizeXS;
+    padding: $euiSizeXS/2 0;
 
     &.index {
       border-left: $euiSizeXS solid $euiColorSuccess;
+
       &.red {
         border-left: $euiSizeXS solid $euiColorDanger;
       }
+
       &.yellow {
```

Comment:
Same here.

Reply:
The classes `index` and `shard` are being passed to the component from outside https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/monitoring/public/components/elasticsearch/shard_allocation/lib/calculate_class.js#L9 I think it would be outside of the scope of this PR to change them.

For `red` and `yellow` I've gone ahead and changed them to `monChild--danger` and `monChild--warning` using a switch statement